pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -658,6 +678,19 @@ public void run() {
     }
   }
 
+  private void checkFastSyncOptions() throws ParameterException {
+    if (fastSyncMaxWaitTime < 0) {
+      throw new ParameterException(
+          commandLine, "--fast-sync-max-wait-time must be a positive or zero integer");
```

Comment:
this range should be explained in the option description too. Otherwise we are going to have issues documenting CLI if we have to read each error case.